test/manydbs/manydbs.c

```diff
@@ -54,6 +54,27 @@ static const char *const uri = "table:main";
 #define MAX_KV 100
 #define MAX_VAL 128
 
+/*
+ * Maximum expected condition variable wakeups. POSIX allows arbitrarily many spurious wakeups to
+ * happen, so we need to be able to adjust this expectation per-platform. There are two cases: when
+ * completely idle, and when running a light workload. The latter is expressed as a fraction of the
+ * total number of condition variable sleeps; the former is a constant.
+ */
+#if defined(__NetBSD__) || defined(_WIN32)
+/*
+ * NetBSD should never generate spurious wakeups, but does: see https://gnats.netbsd.org/56275.
+ * Windows can also generate spurious wakeups:
+ * https://docs.microsoft.com/en-us/windows/win32/sync/condition-variables These values allow the
+ * test to complete in spite of that.
+ */
+#define CV_RESET_THRESH_IDLE 20
```

Comment:
Can you use `THRESHOLD` rather than `THRESH`. Thresh makes me think of processing wheat.

Reply:
done!